app/models/tag_user.rb

```diff
@@ -259,6 +259,7 @@ def self.notification_levels_for(user)
 #
 # Indexes
 #
-#  idx_tag_users_ix1  (user_id,tag_id,notification_level) UNIQUE
-#  idx_tag_users_ix2  (tag_id,user_id,notification_level) UNIQUE
+#  idx_tag_users_ix1                                             (user_id,tag_id,notification_level) UNIQUE
```

Comment:
Is there any reason to keep these indexes as UNIQUE if we introduce a new one for `(user_id, tag_id)`?